themes/_core/js/theme.js

```diff
@@ -22,6 +22,8 @@
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
+__webpack_public_path__ = window.prestashop.core_js_public_path;
```

Comment:
Not sure I get the purpose of this one? Is it to allow using `const zxcvbn = (await import('zxcvbn')).default;` and import from chunks right?

Reply:
Hi @jolelievre,
Webpack need `publicPath` to be set so it can find files that have to be fetched example `publicPath = "/themes/"` - `domain.com/themes/chunkfilename`.
We can set `publicPath` in webpack configuration to just `/themes/` and that should work in 99% but if store `physical_uri` is different than `/` for example `domain.com/store` `publicPath` will be `/store/themes/`. That's why we need that variable to be set on the fly.